Sources/KIF/IdentifierTests/KIFUITestActor-IdentifierTests.h

```diff
@@ -85,6 +85,13 @@
  */
 - (BOOL) tryFindingViewWithAccessibilityIdentifier:(NSString *) accessibilityIdentifier;
 
+/*!
+ @abstract returns YES or NO if the element is visible in the current frame (without scrolling).
+ @discussion if the element described by the accessibility identifier is visible, the method returns true.
+ @param accessibilityIdentifier The accessibility identifier of the element to query for
+ */
+- (BOOL) tryFindingViewInFrameWithAccessibilityIdentifier:(NSString *) accessibilityIdentifier;
```

Comment:
We've been trying to maintain parity between `KIFUITestActor` (the old API) and `KIFUIViewTestActor` (the new API). Would you be willing to add a method like `usingCurrentFrame` or `usingAutomaticScrolling:NO` to `KIFUIViewTestActor`. This would be similar to how `usingTimeout:` works on KIFTestActor to alter matching behavior, but not necessarily altering the predicates used to do the matching. It'd store that state in the actor (defaults to scrolling if not specified) and then pass the scrolling behavior property through [here](https://github.com/kif-framework/KIF/blob/master/Sources/KIF/Classes/KIFUIViewTestActor.m#L610) (and a few lines below). Thoughts?

Reply:
Hey @justinseanmartin, yes it totally makes sense! I'm going to commit these changes today 👌

Reply:
Take a look now 😄

Reply:
Looks great, thanks for the iteration!